docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+version: '3'
+services:
+  mongo:
+    image: mongo:3.6.5
+    volumes:
+      - mongo-data:/data/db
+    ports:
+      - "27017:27017"
+  rabbitmq:
+    image: rabbitmq:3-management
+    hostname: my-rabbit
+    ports:
+      - "5672:5672"
+      - "15672:15672"
+    volumes:
+      - rabbitmq_log:/var/log/rabbitmq
+      - rabbitmq_data:/var/lib/rabbitmq/mnesia/rabbit@my-rabbit
+      - ./config/rabbitmq:/etc/rabbitmq
```

Comment:
I don't think the `rabbitmq` container should be created within this file. People using our gamification service with Docker have to already have an existing RabbitMQ running somewhere. It is not our responsibility (nor should it be) to setup a `rabbitmq` container for our users.

Reply:
to: docker-config for rabbit